sdk/sdk.go

```diff
@@ -33,9 +40,9 @@ type ShannonSDK struct {
 	sessionClient SessionClient
 	accountClient AccountClient
 	paramsClient  SharedParamsClient
-	blockClient   BlockClient
-	relayClient   RelayClient
-	signer        Signer
+	HeightClient
```

Comment:
W/e we end up calling this client. Why is it publically exposed?

s/XClient/xClient

Reply:
Thank you, please see my comment below on the motivation for this.
Additionally, with the latest update to the PR, this particular interface is being dropped.